purchases-js-hybrid-mappings/api-report/purchases-js-hybrid-mappings.api.md

```diff
@@ -0,0 +1,20 @@
+## API Report File for "purchases-js-hybrid-mappings"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+// @public (undocumented)
+export class PurchasesCommon {
+    // (undocumented)

```

Comment:
I don't think it's necessary to add documentation to this class, since it's mostly meant for internal hybrid usage.